src/Host/API/Test/Microsoft.R.Host.Client.API.Test.csproj

```diff
@@ -48,10 +48,6 @@
       <Project>{5EF2AD64-D6FE-446B-B350-8C7F0DF0834D}</Project>
       <Name>Microsoft.UnitTests.Core</Name>
     </ProjectReference>
-    <ProjectReference Include="..\..\API\Impl\Microsoft.R.Host.Client.API.csproj">
-      <Project>{1a02711f-3b13-48be-8d45-073386354174}</Project>
-      <Name>Microsoft.R.Host.Client.API</Name>
```

Comment:
There are issues building new nuget, I will fix this in a separate PR. For now we don't need the new API package anyway.